CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# Enso 2.0.0-alpha.5 (2021-05-14)
+
+<br/>![New Features](/docs/assets/tags/new_features.svg)
+
+#### Visual Environment
+
+#### EnsoGL (rendering engine)
+
+<br/>![Bug Fixes](/docs/assets/tags/bug_fixes.svg)
+
+#### Visual Environment
+
+- [Delete key will delete selected nodes][1538].
+
+#### EnsoGL (rendering engine)
+
+#### Enso Compiler
+
+- [Updated Enso engine to version 0.2.11][1541].
+
+If you're interested in the enhancements and fixes made to the Enso compiler,
```

Comment:
This changelog is not ready to be merged as:
1. it contains unused sections;
2. It does not describe the issues from the user's perspective (it should tell that there was an issue that only backspace was removing nodes and now delete does it as well).